src/pages/tasks/[id].tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import { NextRouter, useRouter } from 'next/router';
+
+import TaskDetails from '@/components/taskDetails';
+
+const TaskDetailsPage: FC = () => {
+  const router: NextRouter = useRouter();
+  const id: any = router.query.id;
+  const TASK_DETAILS_URL: string = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/${id}/details`;
```

Comment:
move this to constants

Reply:
the `id` is coming in dynamically, how do you suggest moving this to constants?

Reply:
` `${TASKS_URL}/${id}/details` `

Reply:
you mean to move `process.env.NEXT_PUBLIC_BASE_URL` to constants or the entire URL?

Reply:
`${TASKS_URL}/${id}/details enclosed in template strings`, if I put this entire thing inside constants, how will I pass the id to it?

Reply:
done